wolfcrypt/src/falcon.c

```diff
@@ -588,9 +588,22 @@ int wc_falcon_export_key(falcon_key* key, byte* priv, word32 *privSz,
  */
 int wc_falcon_check_key(falcon_key* key)
 {
-    /* Might want to try to sign and verify a random message here. */
+    /* Sign and verify a message. */
     int ret = 0;
-    (void)key;
+    int res = 0;
+    byte msg[] = "The wolfSSL team is here to make you ready for quantum computers!!";
```

Comment:
This will have a cast warning on some targets. Please add cast to ‘byte*’. Make ‘const’.